### ISSUE TITLE
Sensor not show

### ISSUE THREAD

**Issue by angeltonio** [gcorgnet/sensor.emby_upcoming_media#11]
I have this..

platform: emby_upcoming_media
api_key: ..... (personal info)
user_id: ..........
host: ....
port: 8096
ssl: false
max: 5
use_backdrop: true
include:
Movies
TV Shows

And the sensor not show on the lovelace...

**Comment by TheRealLouCipher** [comment]
Your include: lookes wrong. Should be like...

````
include:
  - Movies
  - TV Shows

**Comment by angeltonio** [comment]
Yes, I removed the include and it shows me some things, but in theory if that include is removed, should all the libraries come out? Because those with accented letters do not come out

**Comment by christophervonh** [comment]
Hi, I know this issue is old, but maybe it helps anyone else: I had the same issue and my libraries in Emby are named in German (Filme and Serien) so I had to replace _Movies and TV Shows_ in the config file with _Filme and Serien_. For the cards I had to change the default entity: **sensor.emby_latest_movies** to entity: **sensor.emby_latest_filme** (or series) and then it finally worked.
